jest.config.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
   // 1. `react-children-utilities`
   transformIgnorePatterns: ['/node_modules/(?!(react-children-utilities)/)'],
 
-  setupFiles: ['<rootDir>/scripts/jestSetup.js'],
+  setupFiles: ['<rootDir>/scripts/jestSetup.js', 'jest-canvas-mock'],
```

Comment:
Yes. I realized this will break tests for any components that utilize Spinner (i.e. this PR broke Button tests). We're planning to bake Spinner into more components moving forward, so we should probably just set it up globally.